### PR TITLE
Setup styling

### PR DESCRIPTION
**Pull request by matthijsgroen**
- Setup tailwind styling
- Setup theme switcher in storybook
- Add a root 'PassThrough' to style the prime react button as poll app button

### CODE REVIEW

File: packages/components/.storybook/preview.tsx [hunk 1]
```diff
@@ -1,12 +1,33 @@
...
     (Story) => (
-      <PrimeReactProvider>
+      <PrimeReactProvider
+        value={{
+          pt: {
+            button: {
+              root: {
+                className:
+                  "bg-transparent border-2 border-blue-600 hover:border-blue-500 text-blue-600 hover:text-blue-500 border-dashed rounded-none box-shadow-md m-2",
+              },
```
Comment (matthijsgroen) [inline packages/components/.storybook/preview.tsx:26]: This is the styling override of a button, using https://primereact.org/button/ (passthrough tab)

Comment (marcianosr) [reply packages/components/.storybook/preview.tsx:26]: Why does it need these overrides?

Comment (matthijsgroen) [reply packages/components/.storybook/preview.tsx:26]: Without overrides:

image

With overrides:

image

Comment (marcianosr) [reply packages/components/.storybook/preview.tsx:26]: Ah alright!

**Review by marcianosr — APPROVED** [review]

**Pull request merged**